### DISCUSSION TITLE
custom cname config failed

### DISCUSSION THREAD

**Discussion by haomingw** [upptime/upptime#138] [Q&A] [answered]
I set the cname to my custom domain and when I visit [me].github.io/upptime, it is redirected to my custom domain, which is great. But obviously this url has no records in DNS server, which results in a 404. What else should I do to make my custom cname pointed to the upptime status page? Thanks!

**Comment by AnandChowdhary** [comment, marked as answer]
You can follow this to configure the `A` records: https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site

**Reply by haomingw** [reply]
It works, thanks! I added a cname record because it's a custom subdomain. I should have found it on my own but I mal-configured the i18n foot note and had an internal server error.
